blockdev: fail `install --save-partindex` on MBR-formatted disks (#934)
coreos/coreos-installer: docs/release-notes.md

- Add release notes to documentation
- iso: Detect incomplete ISO files
- Warn if console kargs could have used `--console`/`--dest-console` instead

> - install: bail when `--save-partindex` is specified on a MBR disk

Reviewer: The release notes and commit message both refer to `--save-partindex` specifically, but the code also activates if `--save-partlabel` is specified.  We should decide which approach we want to take.

> - install: bail when `--save-partindex` is specified on a MBR disk

Reviewer: ```suggestion
- install: fail if `--save-partindex` is specified on an MBR disk
```

> - save-partindex: bail when listed partindex's are located on a MBR disk  

Reviewer: The prefix should be the subcommand, `install`.  And this implies that we continue if the MBR doesn't contain any of the listed partindexes, which isn't true.  If any partition saving is requested, and the disk is MBR, we bail.

Author: Ah, ok. thank oyu
